modules/special-rules/pages/brood-telepathy.adoc

```diff
@@ -5,6 +5,13 @@ This gives the Cult strong morale.
 
 ---
 
-While at least one Genestealer Patriarch or Magus is present in your Genestealer Cult detachment you automatically pass any Leadership test that you are required to make for them.
+Apply these rules to detachments with the {synapse} ability:
```

Comment:
I've just spotted this after first making a comment below. Shouldn't this say {brood-telepathy}, per the topic title? And what does that mean for the below (I haven't absorbed the related changes enough — no doubt I should look closer another time when I'm in less of a rush. But I'm pretty sure this bit is in error).

Reply:
Yes!
```suggestion
Apply these rules to detachments with the {brood-telepathy} ability:
```